### PR TITLE
Wk 178 post data

### PR DESCRIPTION
**Pull request by aditudorache**
In this PR, post/put data functionality is implemented for the api service.  A small correction in one svg is allso added.

### CODE REVIEW

File: src/components/locationForm/LocationForm.jsx [hunk 1]
```diff
@@ -22,8 +23,13 @@ const LocationForm = ({ id }) => {
...
   });
 
-  const onSubmit = data => {
-    console.log(data);
+  const onSubmit = async data => {
+    try {
+      const url = '/api/location';
+      await sendData(url, data);
+    } catch (error) {
+      // Dispatch the error message. console.log('Error! ', error);
```
Comment (janjaap) [inline src/components/locationForm/LocationForm.jsx]: No error handling?

File: src/shared/api/api.js [hunk 2]
```diff
@@ -58,3 +58,28 @@ export const getByUrl = async (url, params, cancel, reloadOnUnauthorized) => {
     getWithToken(url, params, cancel, token, reloadOnUnauthorized)
   );
 };
+
+const getFormData = data => {
```
Comment (janjaap) [inline src/shared/api/api.js]: IE 11 needs a polyfill for the `FormData` API

File: src/shared/api/api.js [hunk 3]
```diff
@@ -58,3 +58,28 @@ export const getByUrl = async (url, params, cancel, reloadOnUnauthorized) => {
...
+};
+
+export const sendData = async (url, data, method = 'POST') => {
+  const token = getAccessToken();
+  const options = {
+    method,
+    headers: {
+      Authorization: 'Bearer ' + token,
+    },
+    body: getFormData(data),
```
Comment (janjaap) [inline src/shared/api/api.js]: Why not just `JSON.stringify(data)`?

File: src/components/locationForm/LocationForm.jsx (hunk 1 above)
Comment (aditudorache) [reply src/components/locationForm/LocationForm.jsx]: will follow

File: src/shared/api/api.js (hunk 2 above)
Comment (aditudorache) [reply src/shared/api/api.js]: thanks!

File: src/shared/api/api.js (hunk 3 above)
Comment (aditudorache) [reply src/shared/api/api.js]: can't be done is more complex than that, it should provide files for upload

**Pull request merged**